src/pages/CheckoutDetails.jsx

```
@@ -0,0 +1,193 @@
+              key={`cell-${rowIndex}-${cellIndex}`}
+            >
+              {cell}
+            </td>
+          ))}
+        </tr>
+      ))}
+    </tbody>
+  </table>
+);
+
+const BillInfomration = ({title, rows, summary }) => (
```

ESLint

[error] 'title' is missing in props validation

(react/prop-types)